.gitignore

```diff
@@ -5,3 +5,22 @@ js/vendor
 coverage
 npm-debug.log
 package-lock.json
+
+.php_cs.cache
+
+# Composer
+composer.phar
```

Comment:
We should not have ``composer.phar`` "accidentally" loaded from anywhere.
So this should not be needed.